app/controllers/api/message_threads_controller.rb

```diff
@@ -1,4 +1,9 @@
 class Api::MessageThreadsController < Api::TenantController
+  def index
+    @threads = @tenant.message_threads.where('message_threads.id > ?', params[:offset]).order(:id).limit(API_PAGE_SIZE) if params[:offset]
+    @threads = @tenant.message_threads.order(:id).limit(API_PAGE_SIZE) unless params[:offset]
```

Comment:
```suggestion
    @threads = @tenant.message_threads.order(:id).limit(API_PAGE_SIZE)
    @threads = @threads.where('message_threads.id > ?', params[:offset]) if params[:offset]
```

Toto je taky klasicky builder pattern co sa pouziva.

Zaroven offset sa vacsinou pouziva na strankovanie co tuto uplne nie je pravda. Pouzil by som teda radsej explicitne `last_id` ako mame tu https://ekosystem.slovensko.digital/premiove-api#sync-api